### ISSUE TITLE
Add support for widgets

### ISSUE THREAD

**Issue by rakleed** [AdguardTeam/AdGuardVPNForMac#112]
### Problem Description

Sometimes you need to quickly connect to a VPN, but at the same time you don’t want to always keep the application in the background.

### Proposed Solution

Add support for widgets.

### Additional Information

https://developer.apple.com/widgets/

**Comment by Chinaski1** [comment]
Good idea, we have already implemented such a feature in AdGuard AdBlocker, so we will definitely add it to AdGuard VPN.

**Comment by IaroslavKhvorostenko** [comment]
Hello @rakleed 
Thank you for this valuable feature request. It is consistent with our current priorities, and we have scheduled it for a future implementation.
